cooldowns.lua

```diff
@@ -1,87 +1,145 @@
 local _, addon = ...
+local cfg = addon.cfg
 
-local globalCooldown = 1.5
+local LibGlow = LibStub("LibCustomGlow-1.0")
+local decimalThreshold = 3
 
 local function GetAlpha(self, duration)
```

Comment:
This defined four times.

Reply:
I see that's my bad.  The old copy and paste syndrome got me.

Reply:
Consolidated the code.